api/python/arrus/utils/imaging.py

```diff
@@ -2819,7 +2819,7 @@ def prepare(self, const_metadata):
 
         self.tx_foc = self.num_pkg.asarray(focus, dtype=self.num_pkg.float32)
         burst_factor = tx_op.excitation.n_periods/(2 * self.fn)
-        self.initial_delay = -start_sample/65e6+burst_factor+tx_center_delay
+        self.initial_delay = -start_sample/120e6+burst_factor+tx_center_delay
```

Comment:
Sampling frequency should depend on AFE(OEM) variant

Reply:
Fixed